### PR TITLE
chore: release 0.21.0-rc1

### PR DESCRIPTION
**Pull request by Fabien-GELUS**
Signed-off-by: Fabien Gelus <fabien.gelus@owkin.com>

### CODE REVIEW

**Comment by Esadruhn** [comment]
why are there submodules added to the doc?

File: examples/titanic_example/assets/algo_random_forest/predict/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # this base image works in both CPU and GPU enabled environments
-FROM gcr.io/connect-314908/connect-tools:0.15.0-nvidiacuda11.6.0-base-ubuntu20.04-python3.9
+FROM gcr.io/connect-314908/connect-tools:0.16.0-nvidiacuda11.6.0-base-ubuntu20.04-python3.9
```
Comment (Esadruhn) [inline examples/titanic_example/assets/algo_random_forest/predict/Dockerfile:2]: change for the ghcr adress

File: docs/source/conf.py [hunk 2]
```diff
@@ -83,19 +83,27 @@ def __call__(self, directory):
...
-assets_dir_titanic = Path(__file__).parents[2] / "examples" / "titanic_example" / "assets"
+assets_dir_titanic = (
+    Path(__file__).parents[2] / "examples" / "titanic_example" / "assets"
+)
 zip_dir(assets_dir_titanic, "titanic_assets.zip")
 
-assets_dir_substrafl_fedavg = Path(__file__).parents[2] / "substrafl_examples" / "strategies_examples" / "assets"
+assets_dir_substrafl_fedavg = (
+    Path(__file__).parents[2] / "substrafl_examples" / "strategies_examples" / "assets"
+)
```
Comment (Esadruhn) [inline docs/source/conf.py:106]: please avoid reformatting this file automatically
not a big deal, but I think several people have different linter configs on their computer, and this file gets reformatted differently on a regular basis, that's bothersome

**Comment by Esadruhn** [comment]
Will need to update the compatibility table and changelog

**Comment by Fabien-GELUS** [comment]
> compatibility table and changelog

will be done in https://github.com/Substra/substra-documentation/pull/174

**Pull request closed without merging**